<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>歌词下载器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <layout class="QHBoxLayout" name="layoulayou_1" stretch="0,5,1">
        <item>
         <widget class="QLabel" name="songUrlLabel">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>歌曲url：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="songUrlEdit">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>在此输入歌曲URL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <family>Monaco</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>get</string>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="songNameLabel">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>歌名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="songNameEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="singerLabel">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>歌手：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="singerEdit">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="timeCheck">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>时间轴</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="transCheck">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>翻译</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="Lrc">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>520</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
